src/core/server/graph/resolvers/Subscription/commentLeftModerationQueue.ts

```diff
@@ -37,6 +42,22 @@ export const commentLeftModerationQueue: SubscriptionToCommentLeftModerationQueu
         return false;
       }
 
+      // If we're filtering by siteID, then only send back comments from the
+      // specific site.
+      if (siteID && source.siteID !== siteID) {
+        return false;
+      }
+
+      // If we're filtering by section, then only send back comments from the
+      // specific section.
+      if (
+        section &&
+        section.name !== source.section &&
```

Comment:
What about when `section.name === null`? I think `source.section` is `undefined` but never `null`